documentation/production/tutorial-production-mode-basic.asciidoc

```diff
@@ -84,3 +84,102 @@ When minifying code, it is often obscured, which makes it harder to read, hence
 Bundling is an optimization where we merge multiple files to a single collection so that the browser doesn't need to
 request so many files from the server. 
 This makes the application load faster.
+
+== Checklist for the production mode app artifact
+
+Vaadin Servlet configures itself from the `flow-build-info.json` file, which is located in the `META-INF/VAADIN/config/` resource package.
+The location of resources differs for different artifact types, for example:
+
+. Spring Boot jar file places resources into the `BOOT-INF/classes/` folder in the jar file;
+. WAR archive places resources into the `WEB-INF/classes/` folder.
+
+If the production mode has been activated properly, the contents of the `flow-build-info.json` should be as follows:
+
+.flow-build-info.json
+[source, json]
+----
+{
+  "compatibilityMode": false,
+  "productionMode": true,
+  "enableDevServer": false,
+  "chunks": {
+    "fallback": {
+      "jsModules": [
+        "@vaadin/vaadin-icons/vaadin-icons.js",
+        "@vaadin/vaadin-grid/src/vaadin-grid-tree-toggle.js",
+// etc etc
+        "frontend://ironListConnector.js"
+      ],
+      "cssImports": [
+      ]
+    }
+  }
+}
+----
+
+(Note that the `productionMode` property is set to `true`).
```

Comment:
You could use a callout for this (and remove parentheses). // <1>

Reply:
Excellent advice, I will do so :+1: